SubmittingPatches.rst

```diff
@@ -1,3 +1,5 @@
+.. _submitting-patches-to-ceph:
```

Comment:
as explained in my previous comment, this label is not visible by sphinx-build.

Reply:
Understood.  I take this note to mean that you would like it to be removed?  I've pushed that change.